subsys/bluetooth/scan.c

```diff
@@ -1294,19 +1300,25 @@ static void scan_device_found(const bt_addr_le_t *addr, s8_t rssi, u8_t type,
 	/* Check id device is connectable. */
 	if (type == BT_LE_ADV_IND ||
 	    type == BT_LE_ADV_DIRECT_IND ||
-	    type == BT_LE_ADV_DIRECT_IND_LOW_DUTY) {
```

Comment:
why low duty does not set connectable flag?

Reply:
@kapi-no LOW_DUTY is not a PDU type, So it is correct to remove it here.

@KAGA164 SCAN_RSP is not necessarily connectable. Both ADV_SCAN_IND and ADV_IND can be requested for a SCAN_RSP. I would suggest to remove it here, and instead report connectable on the ADV_IND pdu.

Reply:
Ok